streampark-console/streampark-console-service/src/assembly/script/schema/mysql-schema.sql

```diff
@@ -319,6 +319,24 @@ create table `t_team` (
   unique key `team_name_idx` (`team_name`) using btree
 ) engine = innodb default charset = utf8mb4 collate = utf8mb4_general_ci;
 
+-- ----------------------------
+-- Table of t_variable
+-- ----------------------------
+drop table if exists `t_variable`;
+create table `t_variable` (
+  `id` bigint not null auto_increment,
+  `variable_code` varchar(100) collate utf8mb4_general_ci not null comment 'variable code',
+  `variable_value` varchar(1024) collate utf8mb4_general_ci not null comment 'variable value',
+  `variable_name` varchar(100) collate utf8mb4_general_ci not null comment 'variable name',
+  `description` varchar(100) collate utf8mb4_general_ci default null comment 'description',
+  `user_id` bigint collate utf8mb4_general_ci not null comment 'user id',
```

Comment:
The `user_id` is the creator, right?

If yes, could you rename it to creator? It is more clear.

Reply:
Are you referring to comment or fields?